tests/FakeItEasy.Tests/Configuration/StartConfigurationTests.cs

```diff
@@ -1,223 +0,0 @@
-namespace FakeItEasy.Tests.Configuration
-{
-    using System;
-    using System.Linq.Expressions;
-    using FakeItEasy.Configuration;
-    using FakeItEasy.Core;
-    using FakeItEasy.Expressions;
-    using FakeItEasy.Tests;
-    using FakeItEasy.Tests.TestHelpers;
-    using FluentAssertions;
-    using Xunit;
-
-    public class StartConfigurationTests

```

Comment:
Ah, I see now; it's covered in `UnnaturalFakeSpecs`. I didn't know about `StartConfiguration` and didn't realized it was used in unnatural fakes.